src/server/snapshot/snapshotterInjected.ts

```diff
@@ -162,29 +162,29 @@ export function frameSnapshotStreamer(snapshotStreamer: string, snapshotBinding:
       (iframeElement as any)[kSnapshotFrameId] = frameId;
     }
 
-    captureSnapshot(snapshotName?: string) {
-      this._streamSnapshot(snapshotName);
-    }
+    reset() {
+      this._staleStyleSheets.clear();
 
-    setSnapshotInterval(interval: number) {
-      this._interval = interval;
-      if (interval)
-        this._streamSnapshot();
+      const visitNode = (node: Node | ShadowRoot) => {
```

Comment:
In theory, we can replace the cache with WeakMap and just clear it here. Up to you.